packages/expo-asset/src/resolveAssetSource.ts

```diff
@@ -0,0 +1,3 @@
+import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
```

Comment:
Shouldn't we add `react-native` as a peer dependency of `expo-asset`? (Or wrap these requires in try-catch?)

Reply:
Avoid try-catches around requires, they don't work well in client software. We could add the peerDep but it seems like you'd have RN anyway.